src/app/components/admin/regular-table/regular-table.component.css

```
@@ -92,3 +87,31 @@ i:hover {
+.descArrow {
+  transform: rotate(180deg);
+  transition: transform 0.3s ease;
+}
+
+.ascArrow, .descArrow{
+  align-self: center;
+  font-size: large;
+  color: var(--bg-color-primary);
+}
+
+.ascArrow:hover .arrowClicked, .descArrow:hover arrowClicked {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)